save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/component/CvssBaseScoreCalculator.kt

```diff
@@ -247,8 +247,8 @@ val cvssBaseScoreCalculator: FC<CvssBaseScoreCalculatorProps> = FC { props ->
 
                 li {
                     className = ClassName("list-group-item d-flex justify-content-center")
-                    val score = CvssVectorV3(CvssVersion.CVSS_V3_1, baseMetrics).calculateBaseScore()
-                    +"${if (baseMetrics.isValid()) "${getCriticality(score)} $score " else ""}${CvssVectorV3(CvssVersion.CVSS_V3_1,
+                    val score = CvssVectorV3(CvssVersion.CVSS_V3_1, baseMetrics)
+                    +"${if (baseMetrics.isValid()) "${getCriticality(score.calculateBaseScore())} $score.calculateBaseScore() " else ""}${CvssVectorV3(CvssVersion.CVSS_V3_1,
```

Comment:
maybe add this checking to `calculateBaseScore` and return nullable value?

Reply:
Agree, added